src/mapLayers.ts

```diff
@@ -1005,8 +1005,8 @@ export const stewardship = new FeatureLayer({
 export const points = new FeatureLayer({
   title: 'Points',
   id: 'feature-point',
-  url: 'https://wrimaps.utah.gov/arcgis/rest/services/WRI/Features/MapServer/0',
-  fields: ['StatusDescription'],
+  url: 'https://wrimaps.at.utah.gov/arcgis/rest/services/WRI_AT/Features/MapServer/0',
```

Comment:
I assume that this base path will be moved to an env var or something similar at some point?

Reply:
Yes, I'm waiting to see how things play out with the DTS integration.